zingolib/proptest-regressions/wallet/transaction_record.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc 999689fed262cc7fc4511501210291661fdaecacf05e9f6f8913451c8603d943 # shrinks to transparent_unspent = 0, transparent_spent = 7757925406770401872, transparent_semi_spent = 10688818666936149744, sapling_unspent = 0, sapling_spent = 0, sapling_semi_spent = 0, orchard_unspent = 0, orchard_spent = 0, orchard_semi_spent = 0
```

Comment:
Is this because we parameterized a test with something that was too large?  Like a `u64`?

Reply:
removed